mergebot-config.json

```diff
@@ -28,9 +28,9 @@
         "interactive": false,
         "fix_version_map": {
             "master": "DC/OS 1.13.0",
-            "1.12": "DC/OS 1.12.0",
-            "1.11": "DC/OS 1.11.6",
-            "1.10": "DC/OS 1.10.9",
+            "1.12": "DC/OS 1.12.1",
+            "1.11": "DC/OS 1.11.8",
+            "1.10": "DC/OS 1.10.10",
```

Comment:
Does this need to be changed in the 1.12 branch? Or only in `master`? I am almost certain that Mergebot looks only at a single place, and it would be good to know where it looks. 

- If it looks at the `master` branch then these changes won't take effect.

- If it looks at the "corresponding" branch then we don't need the 1.11.x and the 1.10.x changes here.